resources/logging/charts/fluentbit/templates/daemonset.yaml

```diff
@@ -82,6 +82,19 @@ spec:
           mountPath: /secure/forward-tls-ca.crt
           subPath: forward-tls-ca.crt
 {{- end }}
+{{- if and (.Values.backend.http.tls_cert) (.Values.backend.http.tls_key) }}
+        - name: http-tls-secret
+          mountPath: /secure/http-tls.crt
+          subPath: tls.crt
+        - name: http-tls-secret
+          mountPath: /secure/http-tls.key
+          subPath: tls.key
+{{- end }}
+{{- if and (.Values.backend.http.tls_verify) (.Values.backend.http.tls_ca) }}
```

Comment:
Here you make the condition dependent on the verify flag, in the configmap.yaml not. Seems inconsistent, please double-check.

Reply:
It's identical like it was already done for the forward plugin configuration - the same discrepancy can be observed there as well.